include/graphene-config.h.meson

```diff
@@ -21,6 +21,11 @@ extern "C" {
 
 #  if defined(__ARM_NEON__) || defined (_M_ARM64)
 #mesondefine GRAPHENE_HAS_ARM_NEON
+#   if defined (_MSC_VER) && (_MSC_VER < 1920) && defined (_M_ARM64)
```

Comment:
Since this header isn't used anywhere else, could you move this check into the block below?

Reply:
Moved.  I also removed the macro that is defined here and instead include the needed header directly according to the compiler and architecture used.